test/test_supervisor.rb

```diff
@@ -213,6 +214,52 @@ def test_main_process_signal_handlers
     $log.out.reset if $log && $log.out && $log.out.respond_to?(:reset)
   end
 
+  def test_cont_in_main_process_signal_handlers
+    omit "Windows cannot handle signals" if Fluent.windows?
+
+    opts = Fluent::Supervisor.default_options
+    sv = Fluent::Supervisor.new(opts)
+    sv.send(:install_main_process_signal_handlers)
+
+    begin
+      Process.kill :CONT, $$
+    rescue
+    end
+
+    sleep 1
+
+    assert_true File.exist?(@sigdump_path)
+  ensure
+    File.delete(@sigdump_path) if File.exist?(@sigdump_path)
+  end
+
+  def test_term_cont_in_main_process_signal_handlers
+    omit "Windows cannot handle signals" if Fluent.windows?
+
+    create_debug_dummy_logger
+
+    opts = Fluent::Supervisor.default_options
+    sv = Fluent::Supervisor.new(opts)
+    sv.send(:install_main_process_signal_handlers)
+
+    begin
+      Process.kill :TERM, $$
+      Process.kill :CONT, $$
+    rescue
+    end
+
+    sleep 1
+
+    debug_msg = '[debug]: fluentd main process get SIGTERM' + "\n"
+    logs = $log.out.logs
+    assert{ logs.any?{|log| log.include?(debug_msg) } }
```

Comment:
```suggestion
    assert{ logs.any?{|log| log.include?(debug_msg) } }
```

Reply:
It seems to be the same code.
I would like to know if there is a better way.

Reply:
Ah, sorry. This is garbage.